src/integration/pages/mediaAssetPage/pidgin/__snapshots__/canonical.test.js.snap

```diff
@@ -133,24 +133,14 @@ exports[`Canonical Media Asset Page I can see a bulleted list item 1`] = `"<Medi
 
 exports[`Canonical Media Asset Page I can see a bulleted list item with link: <Media Asset Page> 1`] = `"http://www.bbc.com/afrique/23248423"`;
 
-exports[`Canonical Media Asset Page I can see related content: 'I no know say I different for society until pipo begin look me one kain' 1`] = `"/pidgin/media-53580248"`;
```

Comment:
Ffs I think this is coming from updating integration snapshots that are failing on latest. When this is fixed on latest, it'll disappear here.